data.yaml

```diff
@@ -1,7 +1,7 @@
 # The purpose of this config is to keep all versions in a single file and make them machine accessible
 
 # Marks the latest release
-version: "2.12.0"
+version: "2.12.1"
```

Comment:
Please do not bump a version in a bugfix.

Reply:
Ok, I was testing whether e2e is using latest KSM or not.